README.md

```diff
@@ -1 +1,37 @@
-LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
\ No newline at end of file
+# learnaway
+
+LearnAway is a tool to facilitate the self-learning created by two technology enthusiasts, who live far from each other but linked by the passion for programming.
+
+## learnaway-api
+
+### Requirements
```

Comment:
As you wish, really, but the native instructions did not hurt neither. I would have kept both options.